### PR TITLE
improve --one-file-system help /docs

### PR DESCRIPTION
**Pull request by sashadev-sky**
Includes changes from #7489

### CODE REVIEW

**Review by ThomasWaldmann — APPROVED** [review]

**Comment by sashadev-sky** [comment]
@ThomasWaldmann I don't have write access to merge

**Comment by ThomasWaldmann** [comment]
@sashadev-sky no problem, i just wanted to wait until the CI finished.

**Pull request merged**
